test/controllers/claimToAdjustPoa/CheckYourAnswersControllerSpec.scala

```diff
@@ -160,6 +171,16 @@ class CheckYourAnswersControllerSpec extends MockAuthenticationPredicate with Te
         status(result) shouldBe INTERNAL_SERVER_ERROR
         status(resultAgent) shouldBe INTERNAL_SERVER_ERROR
       }
+      "Payment On Account data is missing from session" in {
+        setupTest(
+          sessionResponse = Right(Some(emptySession)),
+          claimToAdjustResponse = poa
+        )
+        val result = TestCheckYourAnswersController.show(isAgent = false)(fakeRequestWithNinoAndOrigin("PTA"))
+        val resultAgent = TestCheckYourAnswersController.show(isAgent = true)(fakeRequestConfirmedClient())
+        status(result) shouldBe INTERNAL_SERVER_ERROR
+        status(resultAgent) shouldBe INTERNAL_SERVER_ERROR
+      }
```

Comment:
* merging Individual and Agent cases is fine within scope of specific case;
* however, I think its worth to test results right away before attempting evaluate results from Agent etc